docs/software/cloud.md

```diff
@@ -532,7 +540,7 @@ This is Mozilla's Firefox Sync Server which manages syncing Firefox instance boo
 
 === "Updating"
```

Comment:
We normally named it "Update to latest version".

Additionally I propose to change the tab order like others to:

1. Configure Firefox
2. Database and config directory
3. Installation directory
4. View logs
5. Update to latest version

Reply:
What do you think about merging the two directory tabs to "Directories", with multiple bullet points inside? A tab for a single path seems like an overuse to me.

Reply:
Went with that, as it was the same in other cases as well: Install dir, data/config dir and major config file in one tab makes sense. It could be renamed to "Used directories" or "Directory structure" or such, but that's another question.

The other you suggest makes sense:
- Quick start/access info first
- Config/tweak/hardening info second
- Troubleshooting/logs/debug info third
- Updates last

Looks to me like the order in which the info becomes relevant.